### PR TITLE
Update SetPropertyValue method for Xamarin.Forms

### PR DESCRIPTION
**Pull request by shyunMin**
This PR is to enhance `SetPropertyValue` to support `Xamarin.Forms` specific properties using `TypeConverter`.

### CODE REVIEW

File: Tizen.Appium.Forms/FormsAdapter.cs [hunk 1]
```diff
@@ -1,19 +1,34 @@
 using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Linq;
```
Comment (ideallee) [inline Tizen.Appium.Forms/FormsAdapter.cs:4]: Do you have any specifics on the order of using packages? I know it's usually sorted alphabetically, but these two are upside down. If the coding rules do not comment on this, it is fine code, but it seems necessary to check if it is mentioned.

**Review by ideallee — COMMENTED** [review]
Please check the using order of the coding rules.

File: Tizen.Appium.Forms/FormsAdapter.cs [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
         {
             return new FormsElementList();
         }
 
         protected override void AdaptApp()
-        {
+        {
+            var asm = typeof(Xamarin.Forms.View).Assembly;
+            var typeinfo = typeof(Xamarin.Forms.TypeConverter).GetTypeInfo();
+            TypeConverters = (from c in asm.DefinedTypes
```
Comment (ideallee) [inline Tizen.Appium.Forms/FormsAdapter.cs:28]: If you clearly indicate what the variable c mentioned here means, the readability can be improved.

**Review by ideallee — COMMENTED** [review]
Please check the readability of the variable.

File: Tizen.Appium.Forms/FormsAdapter.cs (hunk 2 above)
Comment (shyunMin) [reply Tizen.Appium.Forms/FormsAdapter.cs]: Fixed.

File: Tizen.Appium.Forms/FormsAdapter.cs (hunk 1 above)
Comment (shyunMin) [reply Tizen.Appium.Forms/FormsAdapter.cs]: Thank you for reviewing.

**Comment by flyofsky** [comment]
LGTM

**Review by flyofsky — APPROVED** [review]

**Review by ideallee — COMMENTED** [review]
It seems to have been fixed and committed.

**Pull request merged**